src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Tests/Microsoft.Azure.KeyVault.Tests.csproj

```diff
@@ -24,13 +24,6 @@
     <ProjectReference Include="..\Microsoft.Azure.KeyVault.TestFramework\Microsoft.Azure.KeyVault.TestFramework.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <!--<PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0-preview-20170106-08" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0-beta5-build1225" />-->
-    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.2" />
-    <!--<PackageReference Include="xunit" Version="2.2.0-beta5-build3474" />-->
-  </ItemGroup>
-
   <!--<PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
```

Comment:
@dragav please remove commented code from project files, they were there for referencing what the project use to target earlier prior to migration. Once you have everything running, feel freed to clean up all the code that might cause confusion in the project files.